src/com/vinnik/kate/MultithreadingSorter.java

```diff
@@ -0,0 +1,56 @@
+package com.vinnik.kate;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+/** Class, which creates given number of threads, and measures time they need to sort random array. */
+public class MultithreadingSorter {
+
+    /** Creates given number of threads and executes them. */
+    public static void measureTimeOfSorting(int amountOfThreads) {
+        for (int i = 0; i < amountOfThreads; i++) {
+            ThreadSort thread = new ThreadSort(i);
+            thread.start();
+        }
+    }
+
+
+    private static class ThreadSort extends Thread {
+        private int[] numbersToSort;
+        private int key;
+
+        protected ThreadSort(int number) {
+            numbersToSort = new int[500];
+            key = number;
+
+            for (int i = 0; i < numbersToSort.length; i++) {
+                Random random = new Random();
+                numbersToSort[i] = random.nextInt(10000);
+            }
+        }
+
+        public void run() {
+            long startTime = System.nanoTime();
+            sort();
+            long elapsedTimeInNanoseconds = System.nanoTime() - startTime;
+            long elapsedTimeInMilliseconds = TimeUnit.MILLISECONDS.convert(elapsedTimeInNanoseconds, TimeUnit.NANOSECONDS);
+            System.out.println(key + " - " + elapsedTimeInMilliseconds);
```

Comment:
If you'd look at the output, you'd see something like this:
```4 - 1
3 - 1
5 - 4
9 - 0
7 - 0
2 - 7
8 - 0
0 - 9
1 - 9
6 - 2 
```
If I'd be a user, I wouldn't really understand what does this table mean. Please, consider adding measurements.

Reply:
But such print was in description of task, isn't it?